src/coreclr/inc/cordebug.idl

```diff
@@ -7781,6 +7781,17 @@ interface ICorDebugExceptionObjectValue : IUnknown
     HRESULT EnumerateExceptionCallStack([out] ICorDebugExceptionObjectCallStackEnum** ppCallStackEnum);
 };
 
+[
+    object,
+    local,
+    uuid(e3b2f332-cc46-4f1e-ab4e-5400e332195e),
+    pointer_default(unique)
+]
+interface ICorDebugExceptionObjectValue2 : IUnknown
```

Comment:
In other projects it would be common to inherit the non-V2 interface, to avoid unnecessary calls to obtain both interfaces, after getting the newer one. Not sure how the dotnet runtime does this in other cases.

Reply:
Thank you for the feedback. ICorDebug already has an established precedent, so I prefer to maintain consistency.